services/web/src/jest-test/Jest.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from 'react';
```

Comment:
We are leaving this as of now so that the others have some template when they start writing their tests. But it should be removed later down the line